### ISSUE TITLE
Technique works with SharePoint the same as OneDrive for Business

### ISSUE THREAD

**Issue by wynhopkins** [MicrosoftDocs/powerbi-docs#3835]
Under "considerations" the page suggests this won't work for SharePoint,  but it appears to work in exactly the same way as for OneDrive for Business

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b6c1e555-9595-d043-0d5c-fa405c3d0a85
* Version Independent ID: 79411df3-c3d2-3b63-ed5d-aec0b69ed68f
* Content: [Display images in a table or matrix in a report - Power BI](https://docs.microsoft.com/en-us/power-bi/create-reports/power-bi-images-tables)
* Content Source: [powerbi-docs/create-reports/power-bi-images-tables.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/create-reports/power-bi-images-tables.md)
* Service: **powerbi**
* Sub-service: **pbi-reports-dashboards**
* GitHub Login: @maggiesMSFT
* Microsoft Alias: **maggies**

**Comment by wynhopkins** [comment]
Now I've tested it out sharing the reports I can't get it to work for anyone other than the owner of the OneDrive / SharePoint account.

**Comment by maggiesMSFT** [comment]
Hi, @wynhopkins -- would you suggest updating the language in the Considerations section? https://docs.microsoft.com/en-us/power-bi/create-reports/power-bi-images-tables#considerations

**Comment by wynhopkins** [comment]
Hi Maggie

I’ve been trying this out with images from SharePoint and OneDrive (for business), and thought it was working the same on both – hence my original comment.

However, I then realised that when I share the report with someone else it doesn’t work for them regardless of OneDrive or SharePoint and regardless of what sharing options I set.

Is there some tenant setting I need to enable,  or maybe I’m just doing something wrong.

Thanks

Wyn
